drivers/flash/soc_flash_nrf.c

```diff
@@ -174,10 +183,14 @@ static int flash_nrf_write(struct device *dev, off_t addr,
 
 static int flash_nrf_erase(struct device *dev, off_t addr, size_t size)
 {
-	u32_t pg_size = NRF_FICR->CODEPAGESIZE;
+	u32_t pg_size = nrfx_nvmc_flash_page_size_get();
```

Comment:
Is this function missing from the HAL? https://github.com/NordicPlayground/fw-nrfconnect-zephyr/blob/master/ext/hal/nordic/nrfx/hal/nrf_nvmc.h or where is it supposed to be found. I can't find any definition of it in this PR nor in the whole repository. In PR nor master branch so my guess is that some files were not added to this PR.